pkgs/os-specific/linux/power-profiles-daemon/default.nix

```diff
@@ -53,12 +54,16 @@ stdenv.mkDerivation rec {
       pygobject3
       dbus-python
       python-dbusmock
+      pylint
```

Comment:
We tend to skip linters since they are typically meant for development, run by CI and constantly evolving.